tests/many_to_many/tests.py

```diff
@@ -509,6 +509,24 @@ def test_clear(self):
         self.assertQuerysetEqual(self.a4.publications.all(), [])
         self.assertQuerysetEqual(self.p2.article_set.all(), ['<Article: NASA finds intelligent life on Earth>'])
 
+    def test_clear_after_prefetch(self):
+        prefetched_a4 = Article.objects.prefetch_related('publications').get(id=self.a4.id)
```

Comment:
I'd drop the "prefetched_" prefix on the variable name to keep things more readable.